cmd/minishift/cmd/oc_env.go

```diff
@@ -18,25 +18,19 @@ package cmd
 
 import (
 	"fmt"
-
-	"github.com/minishift/minishift/pkg/minikube/constants"
-
 	"os"
 	"path/filepath"
 	"text/template"
 
+	"github.com/minishift/minishift/pkg/minikube/constants"
```

Comment:
If moved, please place this in the common block for all our 'external' imports.,